src/__tests__/useRafLoop.test.tsx

```diff
@@ -0,0 +1,113 @@
+import { act, renderHook } from 'react-hooks-testing-library';
+import useRafLoop from '../useRafLoop';
+
+describe('useRafLoop', () => {
+  it('should be defined', () => {
+    expect(useRafLoop).toBeDefined();
+  });
+
+  it('should call a callback constantly inside the raf loop', done => {
+    let calls = 0;
+    const spy = () => calls++;
+    renderHook(() => useRafLoop(spy), { initialProps: false });
+
+    expect(calls).toEqual(0);
+
+    setTimeout(() => {
+      expect(calls).toBeGreaterThanOrEqual(6);
+      expect(calls).toBeLessThan(10);
```

Comment:
I'm curious if these tests will always work.

Reply:
More or less.
I've decreasse the lower border to 5, o, for most cases - should pass.